coreservices/CatenaX.NetworkServices.sln

```diff
@@ -1,173 +1,163 @@
-		{AFB6DF80-7E9B-4E85-82DF-74686C904ABE}.Release|Any CPU.Build.0 = Release|Any CPU
-		{345AB1AF-8E2E-4669-B468-4F191554E340}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{345AB1AF-8E2E-4669-B468-4F191554E340}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{345AB1AF-8E2E-4669-B468-4F191554E340}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{345AB1AF-8E2E-4669-B468-4F191554E340}.Release|Any CPU.Build.0 = Release|Any CPU
-		{090C46A4-64BD-4512-B027-8AB6E1A56558}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{090C46A4-64BD-4512-B027-8AB6E1A56558}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{090C46A4-64BD-4512-B027-8AB6E1A56558}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{090C46A4-64BD-4512-B027-8AB6E1A56558}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{361B4010-82C0-11EC-8131-8BF7616386B6} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{5BA50973-B032-4C3B-B82F-4136AC6B7DEC} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{06B37DE0-6F10-4D40-97E8-184BA9F6AEA9} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{7A5F84BE-3B45-483A-8EF1-4E1E94F144DC} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{14346800-3A31-4463-91A2-42DCA1E4BFC4} = {06B37DE0-6F10-4D40-97E8-184BA9F6AEA9}
-		{97FE948B-D083-4BE2-9E5A-2AA0A0D7B44D} = {F5E59460-211B-11EC-B6E9-75CD881C6D34}
-		{F5E59460-211B-11EC-B6E9-75CD881C6D34} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{59FC0798-2A3B-4EAA-B4F2-43B1DAB259DB} = {F5E59460-211B-11EC-B6E9-75CD881C6D34}
-		{A1088127-BA35-4B52-9F5A-F5B8FE9DAF18} = {7A5F84BE-3B45-483A-8EF1-4E1E94F144DC}
-		{3911B636-69F0-4B46-944D-499B75A2CB0A} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{FC99059C-95DF-4CBC-9B09-7F4C738E7F2F} = {3911B636-69F0-4B46-944D-499B75A2CB0A}
-		{6703308A-92E1-4101-B63D-B03FC3B73B26} = {5BA50973-B032-4C3B-B82F-4136AC6B7DEC}
-		{591E90F7-9607-4F82-8DF3-0AC657695D4B} = {5BA50973-B032-4C3B-B82F-4136AC6B7DEC}
-		{63F6633F-467A-449E-92E5-145CB8F495A4} = {5BA50973-B032-4C3B-B82F-4136AC6B7DEC}
-		{41E2D910-2D1D-11EC-A4AF-635467519A0C} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{C30D1EC2-2AA6-4802-8988-7972F22BF90F} = {41E2D910-2D1D-11EC-A4AF-635467519A0C}
-		{1E68BC44-0476-43A8-8504-B55B83DACF6F} = {41E2D910-2D1D-11EC-A4AF-635467519A0C}
-		{1A8ADE3E-393B-43D6-BE93-9CDB765CB86E} = {C34FEF41-7935-481B-BD92-BB2A99199BE3}
-		{B472A6B7-8B43-4BDB-98C5-AD76C9D05297} = {1A8ADE3E-393B-43D6-BE93-9CDB765CB86E}
-		{2CEDAD69-E09D-48CA-838B-65D93C26C938} = {361B4010-82C0-11EC-8131-8BF7616386B6}
-		{917F0EB5-8D6F-4819-9607-0682248DFF49} = {361B4010-82C0-11EC-8131-8BF7616386B6}
-		{FE4A5E1E-2CB8-4149-9D22-C8C8D7FF5708} = {361B4010-82C0-11EC-8131-8BF7616386B6}
-		{655B039F-508A-430D-A942-960DDFED465B} = {41E2D910-2D1D-11EC-A4AF-635467519A0C}
-		{AFB6DF80-7E9B-4E85-82DF-74686C904ABE} = {41E2D910-2D1D-11EC-A4AF-635467519A0C}
-		{345AB1AF-8E2E-4669-B468-4F191554E340} = {361B4010-82C0-11EC-8131-8BF7616386B6}
+﻿

```

Comment:
@TobiScw in this file all the line endings were changed, just letting you know. Alright if unavoidable but maybe that doesn't need to be.

Reply:
to avoid merge-conflicts I revertet this avoidable change of line-brakes (switching them back to unix format).